### PR TITLE
docs(workspace): 11ty generator

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,11 +43,14 @@
   },
   "dependencies": {
     "@material/mwc-base": "^0.25.3",
+    "html-entities": "^2.3.2",
+    "jsdom": "^18.1.0",
```
Comment (yinonov) [inline package.json:47]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -4,11 +4,13 @@
   "description": "Vonage's web UI 🎨 toolbelt",
   "main": "index.js",
   "scripts": {
-    "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & web-dev-server --watch",
-    "start": "npm run build:components && npm run build:tokens && web-dev-server",
+    "dev": "eleventy --serve",
+    "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & npm run build:docs -- --serve",
+    "start": "npm run build:components & npm run build:tokens & npm run build:docs -- --serve",
+    "build:docs": "eleventy",
```
Comment (yinonov) [inline package.json:10]:
```suggestion
  "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & web-dev-server --watch",
    "start": "npm run build:components && npm run build:tokens && web-dev-server",
```

File: package.json [hunk 3]
```diff
@@ -4,11 +4,13 @@
   "description": "Vonage's web UI 🎨 toolbelt",
   "main": "index.js",
   "scripts": {
-    "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & web-dev-server --watch",
-    "start": "npm run build:components && npm run build:tokens && web-dev-server",
+    "dev": "eleventy --serve",
```
Comment (YonatanKra) [inline package.json:7]: Eleventy is not our dev. Eleventy is our documentation.  Either `start` or `docs`, but def not `dev` on its own.

File: package.json [hunk 4]
```diff
@@ -4,11 +4,13 @@
...
+    "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & npm run build:docs -- --serve",
+    "start": "npm run build:components & npm run build:tokens & npm run build:docs -- --serve",
+    "build:docs": "eleventy",
     "build:tokens": "sass src/core/theme:dist/core/theme",
     "build:components": "rollup -c config/rollup/rollup.config.prod.ts",
-    "build": "npm run build:components && npm run build:tokens",
+    "build": "npm run build:components && npm run build:tokens && npm run build:docs",
```
Comment (YonatanKra) [inline package.json:13]: This is where I say Nx is needed... too many scripts in the build. It's like we define the dependencies on our own.

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: .eleventy.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  eleventyConfig.addPlugin(codeblockdemo);
+
+
+
+  // const style = {"styles": 'dist/core/theme/*.css'};
+  // eleventyConfig.addPassthroughCopy(style);
+  // eleventyConfig.addWatchTarget(style);
+  // Return your Object options:
```
[the remark this answers is not in the document]
Comment (yinonov) [reply .eleventy.js:21]:
```suggestion
```

File: package.json (hunk 4 above)
Comment (yinonov) [reply package.json:13]: ideally we'll have only build and docs eventually

File: .eleventy.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+// const { EleventyRenderPlugin } = require("@11ty/eleventy");
```
Comment (yinonov) [inline .eleventy.js:1]:
```suggestion
```

File: package.json (hunk 2 above)
Comment (YonatanKra) [reply package.json]: Have you change this back?

File: package.json [hunk 7]
```diff
@@ -4,11 +4,15 @@
...
+    "start": "npm run build:components -- --watch & npm run build:tokens -- --watch & npm run docs:serve",
+    "docs:serve": "eleventy --serve",
+    "docs:build": "eleventy",
     "build:tokens": "sass src/core/theme:dist/core/theme",
     "build:components": "rollup -c config/rollup/rollup.config.prod.ts",
-    "build": "npm run build:components && npm run build:tokens",
+    "build": "npm run build:components & npm run build:tokens",
+    "dev": "eleventy --serve",
+    "start:dev": "npm run build:components -- --watch & npm run build:tokens -- --watch & npm run build:docs -- --serve",
+    "build:docs": "eleventy",
```
Comment (yinonov) [inline package.json:15]:
```suggestion
```

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**